docs/bundle/quick_tour.md

```diff
@@ -69,6 +69,8 @@ $producer->sendEvent('aFooTopic', 'Something has happened');
 // send command to ONE consumer
 $producer->sendCommand('aProcessorName', 'Something has happened');
 ```
+what is "a" prefix above? 'aFooTopic' 'aProcessorName' i need to put this prefix every time?
+first time ever see this prefix in PHP world.
 
```

Comment:
> what is "a" prefix above?

It's prefix

>  need to put this prefix every time

yes. otherwise, it will send all your bitcoins to my wallet

> first time ever see this prefix

:'(

Reply:
@errogaht